File: App/app/src/main/java/com/cinema/ticket_booking/data/model/response/Genre.java
package com.cinema.ticket_booking.data.model.response;
import com.google.gson.annotations.SerializedName;
public class Genre {
    @SerializedName("id")   private int id;
    @SerializedName("name") private String name;
    @SerializedName("slug") private String slug;
    public int getId()      { return id; }
    public String getName() { return name; }
    public String getSlug() { return slug; }
}

File: App/app/src/main/java/com/cinema/ticket_booking/data/model/response/Genre.kt
package com.cinema.ticket_booking.data.model.response

import com.google.gson.annotations.SerializedName

data class Genre(
    @SerializedName("id")
    val id: Int,

    @SerializedName("name")
    val name: String?,

    @SerializedName("slug")
    val slug: String?
)
